src/Arcus.Observability.Telemetry.Core/MessageFormats.cs

```diff
@@ -11,7 +11,7 @@ public static class MessageFormats
         /// <summary>
         /// Gets the message format to log external dependencies; compatible with Application Insights 'Dependencies'.
         /// </summary>
-        public const string DependencyFormat = MessagePrefixes.Dependency + " {@" + ContextProperties.DependencyTracking.DependencyLogEntry + "}";
+        public const string DependencyFormat = "{@" + ContextProperties.DependencyTracking.DependencyLogEntry + "}";
```

Comment:
Can't we get rid of these message-formats, and work with types as explained here 
https://github.com/arcus-azure/arcus.observability/issues/205#issuecomment-833453920

Reply:
Not sure what you mean. We need these constants to retrieve their property value later on during the sinking to Application Insights and the telemetry type filtering.

Reply:
For telemetry type filtering I don't believe we need this since we have the new context property.

For the sinking, I think we could just re-use that and get the main information back based on the name, no?

Reply:
See https://github.com/arcus-azure/arcus.observability/pull/215/files#diff-0defdc0c79223504c82f4ce48ee49faccfb8fdf16640651e1747420d5ded944eR52

Reply:
When you would log as is explained [here](https://github.com/arcus-azure/arcus.observability/issues/205#issuecomment-833453920), you don't need that 'message format' anymore.   In the AI sink, you'd be able to deserialize the string to a type, and retrieve all the properties from that type.

Reply:
OK, now I understand.  This string is used for the 'serialization' type of logging in Serilog:

```csharp
_logger.LogWarning(MetricFormat, metricVariable);
```